client/my-sites/marketing/connections/services-group.jsx

```diff
@@ -130,7 +130,17 @@ const SharingServicesGroup = ( {
 								</Fragment>
 							);
 						}
-
+						{
+							/* Injecting the Fediverse above Twitter */
+							if ( service.ID === 'twitter' && type === 'publicize' ) {
+								return (
+									<>
+										<Components.fediverse key="fediverse" />
+										<Component key={ service.ID } service={ service } />
+									</>
+								);
+							}
```

Comment:
This now results in React "key" warning. The `key` prop here should be passed to the fragment.

```tsx
if ( service.ID === 'twitter' && type === 'publicize' ) {
	return (
		<Fragment key={ service.ID }> // <<<<<<<<< key here
			<Components.fediverse />
			<Component service={ service } />
		</Fragment>
	);
}
```

Reply:
Eventually got back to this in https://github.com/Automattic/wp-calypso/pull/94686 :)